src/diffusers/models/embeddings_flax.py

```diff
@@ -29,7 +29,7 @@ def get_sinusoidal_embeddings(timesteps, embedding_dim):
     embeddings. :return: an [N x dim] tensor of positional embeddings.
     """
     half_dim = embedding_dim // 2
-    emb = math.log(10000) / (half_dim - 1)
+    emb = math.log(10000) / (half_dim - freq_shift)
```

Comment:
@patil-suraj @younesbelkada -> this is why we only got `4e-3` tolerance. We should now get something closer to `1e-4` tolerance.

Reply:
Very nice !! Thanks for spotting the conversion nit 🚀

Reply:
Just tried your fix and the test pass with
`torch.testing.assert_allclose(converted_flax_output, torch_output, rtol=1e-04, atol=1e-04)` 
Thanks again!